<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormPesquisa</class>
 <widget class="QWidget" name="FormPesquisa">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="LblPesquisar">
       <property name="text">
        <string>Pesquisar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="EditValorPesquisa">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnPesquisar">
       <property name="text">
        <string>Pesquisar...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnIncluir">
       <property name="text">
        <string>Incluir</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Dados</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QTableWidget" name="tableResultado"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>EditValorPesquisa</tabstop>
  <tabstop>BtnPesquisar</tabstop>
  <tabstop>BtnIncluir</tabstop>
  <tabstop>tableResultado</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
